packages/datatrak-web/src/api/api.ts

```diff
@@ -7,8 +7,7 @@ import axios from 'axios';
 import FetchError from './fetchError';
 
 // TODO: change this to be the correct default url when orchestration server is ready
-export const API_URL =
-  import.meta.env.REACT_APP_DATATRAK_WEB_API_URL || 'http://localhost:8000/api/v1';
+export const API_URL = import.meta.env.REACT_APP_DATATRAK_WEB_API_URL || 'http://localhost:8100/v1';
```

Comment:
```suggestion
export const API_URL = import.meta.env.REACT_APP_DATATRAK_WEB_API_URL || 'http://localhost:8110/v1';
```
This is not decided as 8110